components/imageproc/src/processor.rs

```diff
@@ -21,6 +22,7 @@ use crate::helpers::get_processed_filename;
 use crate::{fix_orientation, ImageMeta, ResizeInstructions, ResizeOperation};
 
 pub const RESIZED_SUBDIR: &str = "processed_images";
+const TMP_OUTPUT_SUFFIX: &str = ".tmp";
```

Comment:
can we use the tempdir (or tempfile can't remember) crate so the folders get cleaned up automatically?

Reply:
I vanished for a while; sorry! I can refactor this to use [tempfile] if you prefer, yeah.

[tempfile]: https://docs.rs/tempfile/latest/tempfile/

Reply:
Updated! See also: the updated `inotifywait` events in the PR description.

Reply:
The resulting files have kinda odd permissions (`0600`); not sure if Zola cares to change that.